Sources/SettingsFeature/Settings.swift

```diff
@@ -441,6 +441,8 @@ public let settingsReducer = Reducer<SettingsState, SettingsAction, SettingsEnvi
         .receipt
         .originalPurchaseDate
       state.buildNumber = environment.build.number()
+      state.stats.isAnimationReduced = state.userSettings.enableReducedAnimation
+      state.stats.isHapticsEnabled = state.userSettings.enableHaptics
```

Comment:
I think we weren't plumbing haptics into `StatsState`...

Reply:
yeah, I guess really `stats` should prob be optional and we should use a proper routing enum like the other views. that would have forced us to provide the values when drilling down.